src/qtgui/plotter.cpp

```diff
@@ -1936,8 +1936,8 @@ void CPlotter::setNewFftData(const float *fftData, int size)
     if (needIIR) {
         for (int i = 0; i < size; ++i)
         {
-            const double v = m_fftData[i];
-            const double iir = m_fftIIR[i];
+            const float v = m_fftData[i];
+            const float iir = m_fftIIR[i];
```

Comment:
`m_fftData` and `m_fftIIR` are float anyway.

Reply:
I'd leave this one, since v can be much smaller than iir (this is linear scale). I think I did this after seeing some sort of clipping effect.

Reply:
Strange. I wouldn't expect precision issues to come up when multiplying or dividing floats with different magnitudes. Anyway, I'll leave this change out for now and we can come back to it.